docker/Dockerfile.base

```diff
@@ -8,22 +8,21 @@ ENV DEBIAN_FRONTEND=noninteractive
 # If you occasionally need something on the container do
 # sudo apt-get update && apt-get whichever
 
-# hadolint ignore=DL3005,DL3008
 RUN apt-get update && \
     apt-get install --no-install-recommends -y \
-      ca-certificates \
-      curl \
-      iptables \
-      iproute2 \
-      iputils-ping \
-      knot-dnsutils \
-      netcat \
-      tcpdump \
-      net-tools \
-      lsof \
-      linux-tools-generic \
-      sudo \
-   && update-ca-certificates \
-   && apt-get upgrade -y \
```

Comment:
the problem with removing this line is that there is a whole slew of binaries in the base distro which (change) that are subject to CVEs.  We could just re-push the base image every release as that should pick up most CVEs, but I'm not sure that solves the manifest problem you want to solve.

Reply:
I agree @sdake.

Reply:
(I may be agreeing with you.. not sure) I don't think we want `upgrade`. See https://docs.docker.com/develop/develop-images/dockerfile_best-practices/. Instead, we should rely on the `ubunutu:bionic` image to be maintained and upgraded as appropriate? Although the 3 of you have spent much more time looking into this than me :slightly_smiling_face: